### PR TITLE
Podcast show styling

### PR DESCRIPTION
**Pull request by takeller**
- Adds bootstrap JS
- Adds collapsable cards for podcast episodes
- Cleans up podcast show page styling

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -478,4 +478,41 @@ footer ul li a:hover, footer ul li a:focus, footer ul li a:active, footer ul li
 .btn-xl {
   font-size: 11px;
   padding: 15px 45px;
-}
\ No newline at end of file
+}
```
Comment (muydanny) [inline app/assets/stylesheets/application.scss:553]: There we go! This guy knows CSS!

File: app/views/podcasts/show.html.erb [hunk 2]
```diff
@@ -1,25 +1,35 @@
...
 <% @podcast_facade.episodes.each do |episode| %>
-  <section class ="episodes">
-    <p class="title"><%= raw(episode.title).html_safe %></p>
-    <div class="description"><%= raw(episode.description).html_safe %></div>
-    <audio src= <%= episode.audio %> controls> </audio>
-    <p>Released On: <%= episode.pub_date %></p>
-  </section>
+  <div class="card text-white bg-info mb-3 w-75">
+    <div class="card-body">
+      <h5 class="card-title"><%= raw(episode.title).html_safe %></h5>
```
Comment (muydanny) [inline app/views/podcasts/show.html.erb]: Raw and html_safe is saving us.

File: app/javascript/packs/application.js [hunk 3]
```diff
@@ -3,6 +3,10 @@
 // a relevant structure within app/javascript and only use these pack files to reference
 // that code so it'll be compiled.
 
+import "bootstrap"
```
Comment (muydanny) [inline app/javascript/packs/application.js:6]: JavaScript!?!

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (janegreene) [reply app/assets/stylesheets/application.scss:553]: I didn't know he was already full stack?! This is great.

File: app/views/podcasts/show.html.erb [hunk 4]
```diff
@@ -1,25 +1,46 @@
...
+        </button>
+      </p>
+      <div class="collapse" id=<%= "episode#{episode.title.delete(' ')[0..5]}" %>>
+        <div class="card card-body episode-card">
+          <div class="card text-white episode-card mb-3 w-75">
+            <div class="card-body">
+              <h5 class="card-title"><%= raw(episode.title).html_safe %></h5>
+              <p class='card-text'><%= raw(episode.description).html_safe %></p>
+              <audio src= <%= episode.audio %> controls> </audio>
+              <div class="card-footer text-muted episode-footer"><%= episode.pub_date %></div>
```
Comment (kiverso) [inline app/views/podcasts/show.html.erb:40]: good use of collapse to save space for podcasts with long descriptions.

**Pull request merged**